js_modules/dagster-ui/packages/ui-core/src/assets/AssetRecentUpdatesTrend.tsx

```diff
@@ -130,39 +128,42 @@ const OPACITIES: Record<number, number> = {
   4: 0.2,
 };
 
+type EventForPopover =
+  | {
+      __typename: 'AssetLatestInfo';
+      latestRun: null | {
+        id: string;
+        status: RunStatus;
+        startTime: number | null;
+      };
+    }
+  | {__typename: 'FailedToMaterializeEvent'; runId: string; timestamp: string}
+  | {__typename: 'ObservationEvent'; runId: string; timestamp: string}
+  | {__typename: 'MaterializationEvent'; runId: string; timestamp: string};
```

Comment:
We don't need the entire fragments for this type, we just need a small subset of values.